designsafe/apps/api/projects/views.py

```diff
@@ -266,8 +266,10 @@ def post(self, request):
             prj.add_team_members([request.user.username])
 
         # Email collaborators
+        tasks.set_project_id.apply_async(args=[prj.uuid], queue="api")
         tasks.email_collaborator_added_to_project.apply_async(
             args=[
+                prj.project_id,
```

Comment:
Does the `project_id` passed to `email_collaborator_added_to_project` depend on the previous call to `set_project_id`? If so, this looks like a potential race condition.

Reply:
Oh yeah that would be an issue when creating a project for the first time. I'm guessing we could make these tasks into a chain like:
`(tasks.set_project_id.s(...)) | tasks.email_collaborator_added_to_project.s(...)).apply_async()`

Reply:
Another complication: `set_project_id` won't update the `prj.id` value of a project that's already been retrieved from the database, so you'll have to make sure to call `refresh_from_db()` on it.